### PR TITLE
feat(i18n): add missing Polish (and English) translation entries

### PR DESCRIPTION
**Pull request by Glaydus**

### CODE REVIEW

**Comment by Glaydus** [comment]
Adding missing translations (including English).
Minor tweaks to **_SettingsProfileMetadata.vue_** were required.

File: components/settings/SettingsProfileMetadata.vue [hunk 1]
```diff
@@ -66,12 +66,14 @@ const chooseIcon = (i: number, text: string) => {
...
-          type="text" placeholder="Label"
+          type="text" placeholder-text-secondary
+          :placeholder="$t('settings.profile.appearance.profile_metadata_label')"
           input-base
         >
         <input
           v-model="form.fieldsAttributes[i - 1].value"
-          type="text" placeholder="Content"
+          type="text" placeholder-text-secondary
+          :placeholder="$t('settings.profile.appearance.profile_metadata_value')"
```
Comment (patak-dev) [inline components/settings/SettingsProfileMetadata.vue:76]: @userquin would you add a comment to this PR showing how you would like we announce these two new key?

Comment (userquin) [reply components/settings/SettingsProfileMetadata.vue:76]: The message should be this:

New entries to translate (only included in `en-` and `pl-PL`):
- `settings.profile.appearance.profile_metadata_label`
- `settings.profile.appearance.profile_metadata_value`

Comment (userquin) [reply components/settings/SettingsProfileMetadata.vue:76]: We only need to copy the source message and paste it on i18n channel.

**Pull request merged**